day_6/exercises/burrito.js

```diff
@@ -16,11 +16,25 @@ class Burrito {
     this.base = base;
     this.toppings = toppings;
   }
+
+  addTopping(top) {
```

Comment:
Always air on the side of verbosity and clarity when naming. I would say that topping is a much better name than top, for that reason